### PR TITLE
fix(Dialog): Add displayName for dialog footer and content

### PR DESCRIPTION
**Pull request by ymyqwe**
Add displayName to Dialog components to align with other components

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/bf1049c8e38e7b11f6f042fdccf0f16746b4b6b7'>bf1049c8e38e7b11f6f042fdccf0f16746b4b6b7</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by sopranopillow** [comment]
@ymyqwe, Looks like you're missing the change files, you need to run `yarn buildto @fluentui/react` and then `yarn change`

**Comment by ymyqwe** [comment]
> @ymyqwe, Looks like you're missing the change files, you need to run `yarn buildto @fluentui/react` and then `yarn change`

change files added. Which version of @fluentui/react will this change be in?

File: change/@fluentui-react-e393acc7-f1bd-4632-9c6d-359924839807.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "add displayName to dialog footer and content",
```
Comment (sopranopillow) [inline change/@fluentui-react-e393acc7-f1bd-4632-9c6d-359924839807.json:3]:
```suggestion
  "comment": "fix: Add displayName to dialog footer and content",
```

**Comment by sopranopillow** [comment]
> > @ymyqwe, Looks like you're missing the change files, you need to run `yarn buildto @fluentui/react` and then `yarn change`
> 
> @sopranopillow change files added. Which version of @fluentui/react will this change be in?

@ymyqwe, most likely in 8.110.16 or 8.111.0 depending on the type of changes that are waiting for release.

File: change/@fluentui-react-e393acc7-f1bd-4632-9c6d-359924839807.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-e393acc7-f1bd-4632-9c6d-359924839807.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28939&commitId=1735fdaeb732a886ff8a2c3ccd3405b7e60bfcc2&env=prod&clientType=FLUENTUI)


  <details><summary><b>Pivot - Overflow</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Review by sopranopillow — APPROVED** [review]

**Pull request merged**